sdks/python/apache_beam/runners/interactive/interactive_environment.py

```diff
@@ -549,6 +589,50 @@ def set_cached_source_signature(self, pipeline, signature):
   def get_cached_source_signature(self, pipeline):
     return self._cached_source_signature.get(str(id(pipeline)), set())
 
+  def set_dataproc_cluster_manager(self, pipeline):
```

Comment:
In setters it's best to give to the module what should be set. Can you please move out the construction of the cluster manager?

Reply:
Due to a few recent updates to the way I now store these cluster managers, I've shifted this logic into the Interactive Runner.